Integrations/integration-jira.yml

```diff
@@ -229,7 +230,7 @@ script:
             mdObj['description'] = dq(element, 'fields.description');
             mdObj['duedate'] = dq(element, 'fields.duedate');
             mdObj['ticket_link'] = dq(element, 'self');
-            mdObj['creted'] = dq(element, 'fields.created');
+            mdObj['created'] = dq(element, 'fields.created');
```

Comment:
Fixed typo here.. maybe would have been better to put this into an own commit... please give a short feedback if this should be reverted and added to another PR